web-integration/tests/fixtures/rill-cloud-fixtures.ts

```diff
@@ -0,0 +1,83 @@
+import { test as base, type Page } from "@playwright/test";
+import { cliLogin, cliLogout } from "./cli";
+import path from "path";
+import {
+  ADMIN_STORAGE_STATE,
+  VIEWER_STORAGE_STATE,
+  RILL_EMBED_SERVICE_TOKEN_FILE,
+  RILL_ORG_NAME,
+  RILL_PROJECT_NAME,
+  RILL_EMBED_HTML_FILE,
+} from "web-integration/tests/constants";
```

Comment:
Any imports from `web-integration` should read `from "@rilldata/web-integration..."`. (I see this in a few places)